Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.5.2
 ossVersion: 1.5.2
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 6.4.4
+version: 6.4.5
```

Comment:
Please do not bump the chart version. We are happy to accept pull requests but will be managing releases separately at the moment.

Thanks!

Reply:
It was out of habit. :) Reverted.